WBooks/Books_Table/BooksTableViewController.swift

```diff
@@ -50,13 +50,13 @@ class BooksTableViewController: UIViewController {
         
         _view.tableView.register(cell: BookTableViewCell.self)
         
-        booksViewModel.loadBooks()
+        _booksViewModel.loadBooks()
         
-        booksViewModel.books.producer.startWithValues { [unowned self] _ in
+        _booksViewModel.books.producer.startWithValues { [unowned self] _ in
             self._view.tableView.reloadData()
         }
         
-        booksViewModel.errorsSignal.observeValues({ [unowned self] (error) in
+        _booksViewModel.errorsSignal.observeValues({ [unowned self] (error) in
```

Comment:
I think I forgot to tell you this. We talked about the retain cycles and when the signals die and that this will work as we said. But we started adding a good practice for signals and producers to avoid memory leaks when we forget or do not analyze this correctly: always add a `take(duringLifetime: self.reactive.lifetime)` so that you make sure it will die with self in any case.